pkgs/top-level/python-packages.nix

```diff
@@ -441,8 +441,6 @@ in {
 
   ansible-base = callPackage ../development/python-modules/ansible/base.nix { };
 
-  ansible-collections = callPackage ../development/python-modules/ansible/collections.nix { };
```

Comment:
Do we need an alias for this?

Reply:
No, ansible should be its only consumer.